functorch/_src/aot_autograd.py

```diff
@@ -865,7 +869,16 @@ def forward(self, *args, **kwargs):
     return AOTModule()
 
 
-def aot_module_simplified(mod: nn.Module, *top_args, **top_kwargs) -> nn.Module:
+def aot_module_simplified(
+        mod: nn.Module,
+        inputs: List[torch.Tensor],
```

Comment:
The reason you added the inputs is so that you can extract out a fake tensor mode from them. But then... why not just pass in a fake tensor mode then lol.